source/features/action-used-by-link.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+export const isActionPage = (): boolean => location.pathname.split('/', 3).join('/') === '/marketplace/actions';
+
+function init(): void {
+	const actionRepo = select('.d-block.mb-2')!
+		.getAttribute('href')!
+		.replace(location.origin, '')
+		.replace('/', '');
+
+	const actionURL = new URL('https://github.com/search');
```

Comment:
Never include github.com in URLs. The extension must work on GitHub Enterprise
```suggestion
	const actionURL = new URL('search', location.origin);
```